### DISCUSSION TITLE
How to use OpenCV's real-time image display function on Fleet!

### DISCUSSION THREAD

**Discussion by wangbaoyi** [flet-dev/flet#2715] [Q&A]
I want to display an image object in real-time in the Fleet interface, just like OpenCV, but after reading the official documentation, I found no examples!!!

**Comment by FeodorFitsner** [comment]
Are you making an app for desktop or mobile?

**Comment by ndonkoHenri** [comment]
What about this: https://github.com/azu2023/CODES/blob/main/cv_video.py ?
